synthetics-stats/schema.graphql

```diff
@@ -267,3 +267,29 @@ type Transaction @entity {
   blockNumber: Int!
   transactionIndex: Int!
 }
+
+type MarketInfo @entity {
+  id: ID!
+  marketToken: String!
+  indexToken: String!
+  longToken: String!
+  shortToken: String!
+}
+
+type VolumeInfo @entity {
+  id: ID!
+  period: String!
+  volumeUsd: BigInt!
+  marginVolumeUsd: BigInt!
+  swapVolumeUsd: BigInt!
+}
+
+type HourlyVolumeByToken @entity {
```

Comment:
maybe it could be just VolumeByToken and have a period field for consistency and scalability?

do we need tokan A and token B here? 
it's a bit non-intuitively to which volumeUsd token it refers to, and also for position we use tokenA for collateral and tokenB for indexToken which is a bit confusing and need to look at the subgraph code to figure out 

we can just separate SwapsVolumeInfo / PositionsVolumeInfo